augly/utils/expected_output/image_tests/expected_metadata.json

```
@@ -483,6 +483,19 @@
             "src_width": 1920
         }
     ],
+    "skew": [
+        {
+            "dst_height": 1080,
+            "dst_width": 1920,
+            "level": 0.5,
+            "axis": "x-axis",
+            "intensity": 0.5,
```

once you update your intensity function, make sure to update the expected metadata as well if the value changes

---

No Changes basically it takes the minimum value between level and maximum is 2 so the intensity is 0.5 as same as previous !

---

I commented on the intensity function definition -- the output range of the function is [0, 100] not [-inf, 2.0] -- so this number will likely change :)

---

Once you make my suggested change to the intensity function, this number will indeed change.
```suggestion
            "intensity": 25.0,
```